app/src/main/java/com/example/musclenerds/ui/home/HomeFragment.java

```diff
@@ -77,7 +87,15 @@ public void run() {
                     wotdText += "\n- " + exercises.get(i).getReps() + " " + exercises.get(i).getName() + "s";
```

Comment:
This line seems to add an extra 's' to everything pulled from the exercise database. This results in the following behavior on the WOTD:

![image](https://user-images.githubusercontent.com/44413459/113939955-e7e1a200-97b9-11eb-8365-be2b58a98431.png)

Reply:
added a new commit that resolves this: https://github.com/braddeibert/musclenerds/pull/11/commits/77dc006f125663e1635ae4c61c3fa846ed8d7880